locale/zh/messages.json

```diff
@@ -4292,11 +4292,11 @@
       ]
     ]
   },
-  "Filter by Service": {
+  "Filter by Framework": {
     "translation": "按服务筛选",
```

Comment:
We should follow up with our vendor on this and see how to handle stuff like this going forward.  Like if we send this back to them to translate new strings, would they notice we changed the english and update the chinese? Or should we remove the Chinese version to force it to be updated? 

@nLight can you see about getting a meeting or just an email contact with the vendor so we can ask?